README.md

```diff
@@ -28,12 +28,16 @@ az bicep install
 
 ### Deploy
 
+> NOTE: make sure you've run `azd auth login` to login to Azure before proceeding below.
```

Comment:
Let's add this right above the `azd up` command instead of making it a note. No harm in running it if someone is already logged in.